acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob

```diff
@@ -79,15 +79,19 @@ test:
     - ADDONS_TEST=$(./acsoo addons list)
     - echo Testing ${ADDONS_TEST}
     - unbuffer venv/bin/coverage run --branch venv/bin/{{{ odoocmd }}} -c odoo-ci.cfg -d ${DB_NAME} --stop-after-init --no-xmlrpc -i ${ADDONS_TEST} --test-enable | ./acsoo checklog
-    - venv/bin/coverage html
-    - venv/bin/coverage report
+    - venv/bin/coverage html --skip-empty --show-contexts
+    - venv/bin/coverage report --skip-empty
     - if [ "${CI_COMMIT_REF_NAME}" = "master" ] ; then makepot ; fi
   coverage: '/TOTAL\s+\d+\s+\d+\s+\d+\s+\d+\s+(\d+\%)/'
   artifacts:
     expire_in: 1 week
     paths:
       - htmlcov/
+      - coverage.xml
     name: "${CI_PROJECT_NAME}-${CI_JOB_ID}-coverage-html"
+    expose_as: "Test coverage"
```

Comment:
expose_as sounds interesting. Would it make sense for the build artifact too ?

Reply:
I don't think so. We usually download the build artifact on the tag jobs. I don't think we need it on the merge requests.

Reply:
Right, I understand.